packages/ui-source-code-editor/src/SourceCodeEditor/props.ts

```diff
@@ -179,6 +179,10 @@ type SourceCodeEditorOwnProps = {
    */
   attachment?: 'bottom' | 'top'
 
+  height?: string
```

Comment:
I think we should provide a small explanation of the prop, even if it's trivial what it does.

For example in the select component we write for the width the following: "The width of the text input."

Reply:
I've added the descriptions.